src/main/flight/pid.h

```diff
@@ -456,16 +450,22 @@ void applyItermRelax(const int axis, const float iterm,
 void applyAbsoluteControl(const int axis, const float gyroRate, float *currentPidSetpoint, float *itermErrorRate);
 void rotateItermAndAxisError();
 float pidLevel(int axis, const pidProfile_t *pidProfile,
-    const rollAndPitchTrims_t *angleTrim, float rawSetpoint, float horizonLevelStrength, bool newRcFrame);
+    const rollAndPitchTrims_t *angleTrim, float rawSetpoint, float horizonLevelStrength);
 float calcHorizonLevelStrength(void);
 #endif
+
 void dynLpfDTermUpdate(float throttle);
 void pidSetItermReset(bool enabled);
 float pidGetPreviousSetpoint(int axis);
 float pidGetDT();
 float pidGetPidFrequency();
+
 float pidGetFeedforwardBoostFactor();
 float pidGetFeedforwardSmoothFactor();
 float pidGetFeedforwardJitterFactor();
+float pidGetFeedforwardAveraging();
 float pidGetFeedforwardTransitionFactor();
+float pidGetFeedforwardMaxRateLimit();
+float pidGetFeedforwardPidKp(int axis);
```

Comment:
You can replace all these functions with this one:
```suggestion
pidRuntime_t *pidGetPidRuntime(void);
```
This comment makes more sense if you read the long comment in `/main/fc/rc.c`